include/ncrypto.h

```diff
@@ -1734,6 +1753,130 @@ class KEM final {
 
 #endif  // OPENSSL_VERSION_MAJOR >= 3
 
+// ============================================================================
+// AEAD (Authenticated Encryption with Associated Data)
+// Note that AEAD primitives are accessed through the Cipher class instead, if
+// using OpenSSL.
+
+#ifdef OPENSSL_IS_BORINGSSL
```

Comment:
Nit: a comment explaining why this is boringssl specific would be helpful.

Reply:
There is a comment just above, but I'll reword it if it's not clear.